fixtures/releases.json

```diff
@@ -0,0 +1 @@
20200629.3.0","metadata":"https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/32.20200629.3.0/release.json"},{"commits":[{"architecture":"x86_64","checksum":"a3b08ee51b1d950afd9d0d73f32d5424ad52c7703a6b5830e0dc11c3a682d869"}],"version":"32.20200715.3.0","metadata":"https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/32.20200715.3.0/release.json"},{"commits":[{"architecture":"x86_64","checksum":"2579b41aa614c3a40b9e24ff0b9dd288f99222dc3ed3a527ef0d8e8667196ff5"}],"version":"32.20200726.3.1","metadata":"https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/32.20200726.3.1/release.json"},{"commits":[{"architecture":"x86_64","checksum":"902c88022e834fb7998140734fae01b75f78b80f63364e9c5309d62d9a260b7c"}],"version":"32.20200809.3.0","metadata":"https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/32.20200809.3.0/release.json"},{"commits":[{"architecture":"x86_64","checksum":"321d73d8865b2899510461cf16c4c2adb60e70031898c8d89e10eed2ac0e438a"}],"version":"32.20200824.3.0","metadata":"https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/32.20200824.3.0/release.json"},{"commits":[{"architecture":"x86_64","checksum":"b53de8b03134c5e6b683b5ea471888e9e1b193781794f01b9ed5865b57f35d57"}],"version":"32.20200907.3.0","metadata":"https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/32.20200907.3.0/release.json"},{"commits":[{"architecture":"x86_64","checksum":"5090d96f4faf090c90c5f4ebf6c3cdab458acc89f2ba08d1ff704744a143b7e6"}],"version":"32.20200923.3.0","metadata":"https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/32.20200923.3.0/release.json"},{"commits":[{"architecture":"x86_64","checksum":"64bb377ae7e6949c26cfe819f3f0bd517596d461e437f2f6e9f1f3c24376fd30"}],"version":"32.20201004.3.0","metadata":"https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/32.20201004.3.0/release.json"},{"commits":[{"architecture":"x86_64","checksum":"a6f3e91ad6e0b8e132769c1694ed9b747a339c04e4e6256fce3afd81c85e1f1f"}],"version":"32.20201018.3.0","metadata":"https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/32.20201018.3.0/release.json"},{"commits":[{"architecture":"x86_64","checksum":"318de830c2f30a97333cd43aa1d500a46ccfedcb2de70a04d0c48228944346da"}],"version":"32.20201104.3.0","metadata":"https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/32.20201104.3.0/release.json"},{"commits":[{"architecture":"x86_64","checksum":"cad80088392fe43bd3cadf0481c3267f199afa7d9f83bc03937ffdbf5ebbc6da"}],"version":"33.20201201.3.0","metadata":"https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/33.20201201.3.0/release.json"},{"commits":[{"architecture":"x86_64","checksum":"a64854cbcec13e1c3b3ccbfd3802e377e23c0c136d384c32736addf77e0e2a03"}],"version":"33.20201214.3.1","metadata":"https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/33.20201214.3.1/release.json"}],"metadata":{"last-modified":"2021-01-05T10:15:52Z"},"stream":"stable"}
```

Comment:
We're going to have to change the test fixture every time the output changes, so let's use `-pretty-print` to make the diffs readable.

Reply:
We're already piping through `jq` to remove the timestamp which pretty prints by default.

Reply:
Oh sorry, you were talking about `releases.json`.  Fixed.